sdk/sas-auth-browser/README.md

```diff
@@ -80,6 +80,12 @@ Launches a popup window that navigates to the authentication endpoint and allows
 
 rejects: When login fails (for example: user closes the popup).
 
+##### `logout(): Promise<void>`
+
+Makes an endpoint call to the Viya server that ends the cookie based browser session.
+
+rejects: When logout fails.
+
```

Comment:
Doc for new api

Reply:
For legal reasons, we need to use "SAS Viya server" on line 85.
The VA UI uses "Sign out" (which matches the standard SAS sign in/sign out window). It seems like line 87 should use either of these: "When signing out fails." Or "When sign out fails."